docs/application/dotnet/guides/nui/nui-components/Notification.md

```diff
@@ -42,7 +48,27 @@ To create a notification using property, follow these steps:
     notification.Post(3000);
     ```
 
-3. Respond to OnWindowTouch:
+3. Set notfication content:

```

Comment:
Corrected notification spell & introduced "the" article
```suggestion
3. Set the notification content:
```

Reply:
change done, thank you for your comment